rubix-spi/src/main/java/com/qubole/rubix/spi/CacheConfig.java

```diff
@@ -78,6 +78,7 @@
   private static final String KEY_DUMMY_CLUSTER_MANAGER = "rubix.dummy.clustermanager.class";
   private static final String KEY_ENABLE_FILE_STALESSNESS_CHECK = "rubix.enable.file.staleness-check";
   private static final String KEY_STALE_FILEINFO_EXPIRY_PERIOD = "rubix.stale.fileinfo.expiry.period";
+  private static final String KEY_CLEANUP_FILES_DURING_START = "rubix.cleanup.files.duing.start";
```

Comment:
Typo! I think the configuration you want to name is "rubix.cleanup.files.**during**.start" instead of "rubix.cleanup.files.**duing**.start"